hummingbot/strategy/cross_exchange_market_making/start.py

```diff
@@ -63,17 +64,17 @@ def start(self):
     )
     self.strategy = CrossExchangeMarketMakingStrategy(
         market_pairs=[self.market_pair],
-        min_profitability=min_profitability,
+        min_profitability=Decimal(repr(min_profitability)),
         status_report_interval=strategy_report_interval,
         logging_options=strategy_logging_options,
-        order_amount=order_amount,
+        order_amount=Decimal(repr(order_amount)),
```

Comment:
Again, if we add a "Decimal" type to order helpers conversion, then we won't need to do this.